perception/object_merger/include/object_association_merger/node.hpp

```diff
@@ -67,11 +68,11 @@ class ObjectAssociationMergerNode : public rclcpp::Node
   std::unique_ptr<DataAssociation> data_association_;
   std::string base_link_frame_id_;  // associated with the base_link frame
   bool remove_overlapped_unknown_objects_;
-  double distance_threshold_;
   struct
   {
     double precision_threshold;
     double recall_threshold;
+    std::vector<double> distance_threshold_list;
```

Comment:
It maybe better to use std::map.
```
std::map<int /*class label*/, double /*distance_threshold*/> distance_threshold_list;
```

Reply:
Use map in https://github.com/autowarefoundation/autoware.universe/pull/1902/commits/1bddcde845419318db542442f56e0c860643d952